src/view/gui/fMainWindow.h

```diff
@@ -1563,7 +1563,7 @@ public slots:
   Fetalbrain mfetalbrain;
 
   fHelpDialog* mHelpDlg;
-  fHelpTutorial mHelpTutorial;
+  fHelpTutorial* mHelpTutorial;
```

Comment:
Does this not need to get destroyed?

Reply:
Good catch, normally I would parent these. But they act really really strangely when I do so.
Since they only really live until the application closes, and no others are created, this doesn't cause a leak in any practical sense.
Still, I just added a change that calls for its destruction in the destructor of fMainWindow.